<ui version="4.0" >
 <class>WalkSiteWndExClass</class>
 <widget class="QWidget" name="WalkSiteWndExClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>101</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Status: Complete Page:%1 Complete Size:%2 Average Speed:%3</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="listView" />
       </item>
       <item>
        <widget class="QListView" name="listView_2" />
       </item>
       <item>
        <widget class="QWidget" native="1" name="radar_widget" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="5" >
           <widget class="QPushButton" name="wsw_pb_start_walk" >
            <property name="text" >
             <string>&amp;Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" >
           <widget class="QPushButton" name="pushButton" >
            <property name="text" >
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QSpinBox" name="wsw_sb_recursive_level" >
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Max Recursive Level&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Thread Count:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2" >
           <widget class="QSpinBox" name="wsw_sb_max_size" >
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Max Size(MB)&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="maximum" >
             <number>1024</number>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5" >
           <widget class="QSpinBox" name="wsw_sb_thread_count" >
            <property name="toolTip" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Max Thread Count&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="maximum" >
             <number>50</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Max Size(MB):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Recursive Level:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Save To:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="wsw_cb_save_to" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>7</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable" >
               <bool>true</bool>
              </property>
              <item>
               <property name="text" >
                <string>.</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="wsw_tb_show_save_path" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Base URL:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="wsw_le_base_url" >
              <property name="text" >
               <string>http://localhost/</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="wswe_setting_menu_button" >
     <property name="windowModality" >
      <enum>Qt::NonModal</enum>
     </property>
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>icons/crystalsvg/32x32/actions/previous.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoRepeat" >
      <bool>false</bool>
     </property>
     <property name="autoExclusive" >
      <bool>false</bool>
     </property>
     <property name="autoRaise" >
      <bool>false</bool>
     </property>
     <property name="arrowType" >
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Span_Hosts" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Span Hosts</string>
   </property>
  </action>
  <action name="action_Follow_Ftp" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Follow Ftp</string>
   </property>
  </action>
  <action name="action_TimeStamp" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>TimeStamp</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="text" >
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRestart" >
   <property name="text" >
    <string>Restart</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionLoad" >
   <property name="text" >
    <string>Load ...</string>
   </property>
  </action>
  <action name="actionHidden" >
   <property name="text" >
    <string>Hidden</string>
   </property>
  </action>
  <action name="actionShow_Radar_Scanner" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Radar Scanner</string>
   </property>
  </action>
  <action name="actionOpen_Directory" >
   <property name="text" >
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionSetting" >
   <property name="text" >
    <string>Setting...</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionFloating" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Floating</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
